artpaint/paintwindow/PaintWindow.cpp

```diff
@@ -130,57 +131,31 @@ PaintWindow::PaintWindow(BRect frame, const char* name, uint32 views,
 		fAdditionalHeight += fMenubar->Bounds().Height() + 1.0;
 	}
 
-	BRect bounds = Bounds().InsetByCopy(-1.0, -1.0);
-	bounds.top = fMenubar ? fMenubar->Bounds().Height() : bounds.top;
-
-	fHorizontalScrollbar = new BScrollBar(BRect(bounds.left,
-		bounds.bottom - B_H_SCROLL_BAR_HEIGHT,
-		bounds.right - B_V_SCROLL_BAR_WIDTH, bounds.bottom),
-		"horizontal", NULL, 0, 0, B_HORIZONTAL);
-	AddChild(fHorizontalScrollbar);
+	fHorizontalScrollbar = new BScrollBar(BRect(0, 0, 0, 0), "horizontal", NULL, 0, 0, B_HORIZONTAL);
```

Comment:
Something to look out for: if there's a BRect in some control's c'tor, consult the HaikuBook if there's a layout aware version. As is the case for BScrollBar.